### ISSUE TITLE
Newsletter component

### ISSUE THREAD

**Issue by harshmangalam** [harshmangalam/event-blend-frontend#24]
- Create new subscribe newletter component
- Add email field and subscribe button
- Use the component just above the footer in Home page.

Do not worry about api integration will work on that later.

**Comment by Anushlinux** [comment]
Hey I am a ts dev, I would like to work on this issue can you assign it to me?

**Comment by harshmangalam** [comment]
Thanks @Anushlinux  assigned to you

**Comment by harshmangalam** [comment]
@Anushlinux any updates on this this issues ?

**Comment by Anushlinux** [comment]
> @Anushlinux any updates on this this issues ?

Yes I am having my exams right on will get back to it in a few days max

**Comment by harshmangalam** [comment]
Okay, please focus on exams you can start after that.

**Comment by HP200421** [comment]
Hey is this issue is still pending.
I would like to work upon it

**Comment by harshmangalam** [comment]
Assigned to you @HP200421

**Comment by HP200421** [comment]
@harshmangalam Thanks
I will be soon working on both the issues
